### PR TITLE
Mcs 27 Generate Scenes with Many Different Player Starting Locations

### PR DESCRIPTION
**Pull request by chrislong**
Implements MCS-27

### CODE REVIEW

File: scene_generator/scene_generator.py [hunk 1]
```diff
@@ -28,38 +30,69 @@
...
 
 def generate_file(name):
-    global DEFAULT_JSON
-    body = json.loads(DEFAULT_JSON)
+    global OUTPUT_TEMPLATE
+    body = copy.deepcopy(OUTPUT_TEMPLATE)
     body['name'] = os.path.basename(name)
+    position = body['performerStart']['position']
+    position['x'] = random_position()
+    position['y'] = random_position()
```
Comment (wagnerwj) [inline scene_generator/scene_generator.py]: This should be position['z']

File: scene_generator/scene_generator.py [hunk 2]
```diff
@@ -28,38 +30,69 @@
...
+    body['performerStart']['rotation']['y'] = random_rotation()
```
Comment (wagnerwj) [inline scene_generator/scene_generator.py]: shouldn't this me position['rotation']['y']?

File: scene_generator/scene_generator.py (hunk 1 above)
Comment (chrislong) [reply scene_generator/scene_generator.py]: Oops, yes, it sure should. I'll push a fix. Thanks.

File: scene_generator/scene_generator.py (hunk 2 above)
Comment (chrislong) [reply scene_generator/scene_generator.py]: I don't think so. From what I've seen, I'm pretty sure `rotation` is part of `performerStart`, not `position`.

Comment (ThomasSchellenbergNextCentury) [reply scene_generator/scene_generator.py]: Chris, you are correct.  In Unity, `position`, `rotation`, and `scale` are three separate objects (all under an object's `transform` property).

Comment (ThomasSchellenbergNextCentury) [reply scene_generator/scene_generator.py]: You don't have to worry about `transform` here though; I just mentioned it to illustrate that Unity manages `position` and `rotation` separately.

**Pull request merged**